sdks/java/core/src/main/java/org/apache/beam/sdk/values/Row.java

```diff
@@ -537,7 +559,7 @@ static int deepHashCodeForMap(
     }
 
     static boolean deepEqualsForCollection(
-        Collection<Object> a, Collection<Object> b, Schema.FieldType elementType) {
+            Collection<Object> a, Collection<Object> b, Schema.FieldType elementType) {
```

Comment:
I think unnecessary whitespace changes like this are what's making our CI unhappy. Can you run spotless to format the code? `./gradlew spotlessApply`

You might also double check there aren't other checkstyle failures: `./gradlew :sdks:java:extensions:sql:checkstyleMain :sdks:java:extensions:sql:checkstyleTest`

Reply:
done